src/app/[locale]/incident/add/components/IncidentQuestionsLocationForm.tsx

```diff
@@ -32,12 +28,18 @@ export const IncidentQuestionsLocationForm = () => {
   useEffect(() => {
     const appendAdditionalQuestions = async () => {
       try {
-        const additionalQuestions = (await fetchAdditionalQuestions(
-          formStoreState.main_category,
-          formStoreState.sub_category
-        )) as unknown as PublicQuestion[]
+        if (
+          formStoreState.main_category !== 'overig' &&
+          formStoreState.sub_category !== 'overig'
+        ) {
+          const additionalQuestions = (await fetchAdditionalQuestions(
+            formStoreState.main_category,
+            formStoreState.sub_category
+          )) as unknown as PublicQuestion[]
```

Comment:
Op zich is dit prima om het hier te doen, mooier is misschien nog om de type casting al binnen de functie fetchAdditionalQuestions te doen. Je abstraheert dan daar al de "imperfecties" van de API spec weg.